wrapper/pre_copy.py

```diff
@@ -202,7 +204,13 @@ def auth_cb(cred, _):
 
         cred_info = [[libvirt.VIR_CRED_AUTHNAME, libvirt.VIR_CRED_PASSPHRASE],
                      auth_cb, None]
-        conn = libvirt.openAuth(self._uri, cred_info)
+
+        if re.compile('^ssh://').match(self._uri):
+            uri = "esx://%s@%s/?no_verify=1" % (self.user, self.server)
```

Comment:
So far we have been using `insecure_connection` to disable SSL verification only for output connections, but it may make sense to extend this also to input connection here.

Alternatively we can keep requesting `no_verify` in input URI. You can then check if it is present in the query part of the input URI and propagate it here.

Reply:
The input URI for SSH doesn't have `no_verify` parameter.

Reply:
I don't think there's any specification around that. Unless it breaks virt-v2v (in case it accidentally sees it) I don't see a reason why we couldn't add that at our end.